pointcloud/src/main/scala/geotrellis/pointcloud/spark/triangulation/BoundaryDelaunay.scala

```diff
@@ -383,17 +454,27 @@ object BoundaryDelaunay {
       val newBound: ResultEdge = copyConvertBoundingLoop()
       var e = dt.boundary
       var ne = newBound
+      //navigateThis(ne)
+
       val boundingTris = collection.mutable.Set.empty[Int]
       do {
         // println(s"in CCBT $e")
+        assert(getDest(e) == halfEdgeTable.getDest(ne) && getSrc(e) == halfEdgeTable.getSrc(ne))
+        // if (circumcircleLeavesExtent(boundingExtent)(getFlip(e))) {
+        //   val f = getFlip(e)
+        //   println(s"Triangle ${(getSrc(f), getDest(f), getDest(getNext(f)), getDest(getNext(getNext(f))))} has circumcircle outside extent")
+        // }
         recursiveAddTris(getFlip(e), ne)
+        //navigateThis(ne)
         e = getNext(e)
         ne = halfEdgeTable.getNext(ne)
       } while (e != dt.boundary)
 
+      writeWKT("bounds.wkt")
       //isMeshValid(triangles, halfEdgeTable)
 
       fillInnerLoop
+      writeWKT("filled.wkt")
```

Comment:
should these `writeWKT` calls be here? O: or we need to cleanup that code (as there are lots of commented out other lines), or that would be a sep issue?